hbase-thrift/src/main/java/org/apache/hadoop/hbase/thrift2/client/ThriftTable.java

```diff
@@ -410,8 +410,7 @@ public boolean thenMutate(RowMutations mutation) throws IOException {
   }
 
 
-  @Override
-  public boolean checkAndMutate(byte[] row, byte[] family, byte[] qualifier, CompareOperator op,
+ private boolean checkAndMutate(byte[] row, byte[] family, byte[] qualifier, CompareOperator op,
```

Comment:
Why does this one get private?

Reply:
It is called by other methods in this class so we can not just remove it...